tests/macsec/test_macsec_protocol_shutdown.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+import logging
+import json
+import time
+import sys
+import os
+
+from .macsec_config_helper import enable_macsec_feature, disable_macsec_feature
+from .macsec_helper import get_mka_session, get_sci, check_mka_session
+from .macsec_platform_helper import get_macsec_ifname
+
+if sys.version_info.major > 2:
+    from pathlib import Path
+    sys.path.insert(0, str(Path(__file__).parent))
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.macsec_required,
+    pytest.mark.topology("any")
+]
+
+profile_name = "256_XPN_SCI"
+
+
+@pytest.fixture(scope='module')
+def setup(duthost, macsec_nbrhosts):
+    dut_num_asics = duthost.num_asics()
+
+    with open(os.path.dirname(__file__) + '/profile.json') as f:
+        macsec_profiles = json.load(f)
+        macsec_profile = macsec_profiles[profile_name]
+        macsec_profile["name"] = profile_name
+
+    nbrhosts = []
+    for nbr in macsec_nbrhosts:
+        nbrhosts.append(macsec_nbrhosts[nbr]["host"])
+
+    setup_info = {
+        'duthost': duthost,
+        'dut_num_asics': dut_num_asics,
+        'nbrhosts': nbrhosts,
+        'macsec_profile': macsec_profile
+    }
+
+    logger.info('Setup_info: {}'.format(setup_info))
+
+    yield setup_info
+
+
+def test_macsec_protocol_shutdown(duthost, macsec_nbrhosts, setup, ctrl_links):
```

Comment:
You should not directly use duthost feature here and in the definition of setup fixture because this may get you a T2 supervisor when the test is applied to a T2 testbed.